File: taskassigningapp.client/src/app/core/models/task/taskDto.ts
export class TaskDto{
  public id: string; 
  public title: string;
  public difficulty: number;
  public type: string;
  public status: string;
  public assignToUser: string | undefined;

  public deadline: Date = new Date();
  public description: string = '';
  public services: string = '';
  public servers: string = '';

  constructor(id: string, title: string, difficulty: number, type: string, status: string, assignToUser: string | undefined){
    this.id = id;
    this.title = title;
    this.difficulty = difficulty;
    this.type = type;
    this.status = status;
  }
}

File: TaskAssigningApp.Server/DTOs/TaskDto.cs
namespace TaskAssigningApp.Server.DTOs
{
    public class TaskDto
    {
        public string Id { get; set; }
        public string Title { get; set; }
        public int Difficulty { get; set; }
        public string Type { get; set; }
        public string Status { get; set; }
        public string? AssignedToUserId { get; set; }

        //DeploymentTask
        public DateTime Deadline { get; set; }
        //ImplementationTask
        public string Description { get; set; }
        //MaintenanceTask
        public string Services { get; set; }
        public string Servers { get; set; }


    }
}
